lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/domain/request/dvad/Input.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.di.ipv.cri.passport.checkpassport.domain.request.dvad;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.di.ipv.cri.passport.library.config.GlobalConstants;
+import uk.gov.di.ipv.cri.passport.library.domain.PassportFormData;
+
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Input {
+
+    private String passportNumber;
+    private String nationalityCode;
+    private String forenames;
+    private String surname;
+    private String dateOfBirth;
+    private String dateOfIssue;
+    private String dateOfExpiry;
+    private String placeOfBirth;
+    private String gender;
+    private String previousSurname;
+    private String bookType;
+
+    public Input(PassportFormData formData) {
+        passportNumber = formData.getPassportNumber();
+        nationalityCode = GlobalConstants.UK_ICAO_ISSUER_CODE;
+        forenames = String.join("", formData.getForenames());
```

Comment:
Are we concatenating forenames here should this not be String.join(" ", formData.getForenames());

Reply:
Fixed, this is a bug and would have broken for users who entered a middlename